client/src/components/Reports/index.js

```diff
@@ -0,0 +1,358 @@
+                          >
+                            {options.children}
+                          </Paper>
+                        </Popper>
+                      )}
+                    />
+                  </div>
+                </div>
+                <div className={classes.block}>
+                  <div className={classes.stockInfo}>
+                    <Typography variant="h6">
+                      Price: ${data.length ? data[data.length - 1].close : ""}
+                    </Typography>
+                    <Typography variant="h6" style={{ marginLeft: "50px" }}>
+                      Change:{" "}
+                      {data.length
+                        ? this.changePoints(
+                            data[data.length - 1].close,
+                            data[data.length - 1].open
+                          )
+                        : ""}
+                    </Typography>
+                  </div>
+                  <div className={classes.stockInfo}>
+                    <Typography variant="h6">
+                      Volume:{" "}
+                      {data.length
+                        ? this.shortenVolume(data[data.length - 1].volume)
+                        : ""}
+                    </Typography>
+                    <Typography variant="h6" style={{ marginLeft: "50px" }}>
+                      Change %:{" "}
+                      {data.length
+                        ? this.changePercent(
+                            data[data.length - 1].close,
+                            data[data.length - 1].open
+                          )
+                        : ""}
+                    </Typography>
+                  </div>
+                </div>
+              </Grid>
+              <Grid item xs={12} md={12}>
+                <Paper
+                  className={classes.paper}
+                  style={{ position: "relative" }}
+                >
+                  <AppBar position="static">
+                    <Tabs
+                      value={value ? value : 0}
+                      onChange={this.handleTabChange}
+                      indicatorColor="primary"
+                      textColor="secondary"
+                      variant="scrollable"
+                      scrollButtons="auto"
+                    >
+                      <Tab label="Price" />
+                      <Tab label="Moving Average" />
+                      <Tab label="Moving Average Convergence" />
+                      {/* <Tab label="Average True Range" />
+                      <Tab label="Volume Weighted Average" /> */}
+                    </Tabs>
+                  </AppBar>
+                  {value === 0 && (
+                    <TabContainer>
+                      <PriceChart data={this.state.data} />
+                    </TabContainer>
+                  )}
+                  {value === 1 && (
+                    <TabContainer>
+                      <MAChart data={this.state.data} />
```

Comment:
Date and EMA is crossing into the candle stick charts. Date is also showing 0NaN-NaN-NaN not sure if that is fixable but otherwise looks good